acceptance/setup/packages/pre-suite/010_Install.rb

```diff
@@ -48,5 +48,15 @@
 install_packages_on(master, MASTER_PACKAGES)
 install_packages_on(agents, AGENT_PACKAGES)
 
+agents.each do |agent|
+  if agent['platform'] =~ /windows/
+    arch = agent[:ruby_arch] || 'x86'
+    base_url = "http://builds.puppetlabs.lan/puppet/#{ENV['SHA']}/artifacts/windows"
+    filename = "puppet-agent-#{ENV['VERSION']}-#{arch}.msi"
```

Comment:
@melissa @haus @Iristyle I'm wondering if it would instead be better for the MSI build automation to create an MSI whose version is full `git describe`, e.g. `3.7.4-1303-ge96f55b`, instead of stripping the short sha, e.g. `3.7.4-1303`? I ask because I imagine the latter is something we're already passing through the AIO pipeline, maybe?